x/provider/ante/ante.go

```diff
@@ -44,23 +44,23 @@ func (dd providerDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool
 
 func (dd providerDecorator) handleMsgCreate(ctx sdk.Context, msg types.MsgCreateProvider) error {
 	if _, ok := dd.keeper.Get(ctx, msg.Owner); ok {
-		return errors.Wrapf(keeper.ErrProviderAlreadyExists, "id: %s", hex.EncodeToString(msg.Owner))
+		return errors.Wrapf(types.ErrProviderExists, "id: %s", hex.EncodeToString(msg.Owner))
```

Comment:
`hex.EncodeToString()` isn't needed here and actually makes the output less useful.  see the others below.